example/tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react",
+    "noFallthroughCasesInSwitch": true
```

Comment:
Just double checking, it sounds like @suzubara is aware and okay with this change being in the PR. Is that the case?

Reply:
Yay! Yes, Suz is aware, here's the [Slack conversation](https://trussworks.slack.com/archives/C0122MPMXL3/p1612301170007100)